components/SettingsControls/samples/SettingsExpanderDragHandleSample.xaml

```diff
@@ -0,0 +1,72 @@
+    -->
+    <ListView AllowDrop="True"
+              CanDragItems="True"
+              CanReorderItems="True"
+              ItemsSource="{x:Bind MyDataSet}"
+              SelectionMode="None">
+        <ListView.ItemTemplate>
+            <DataTemplate x:DataType="local:ExpandedCardInfo">
+                <Grid>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto" />
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+                    <!--  Provide a custom area that can be manipulated by the user  -->
+                    <!--  TODO: Show how to highlight this on hover with a behavior  -->
+                    <Border Width="18"
+                            Margin="1,1,0,1"
+                            Padding="0"
+                            VerticalAlignment="Stretch"
+                            Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+                            CornerRadius="{ThemeResource ControlCornerRadius}">
+                        <TextBlock HorizontalAlignment="Center"
+                                   VerticalAlignment="Center"
+                                   FontFamily="Segoe UI Symbol"
+                                   FontSize="16"
+                                   Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                                   Opacity="0.5"
+                                   Text="&#x283F;" />
+                    </Border>
+                    <controls:SettingsExpander Grid.Column="1"
+                                               Description="{x:Bind Info}"
+                                               Header="{x:Bind Name}">
+
+                        <ToggleSwitch OffContent="Off"
+                                      OnContent="On" />
+
+                        <controls:SettingsExpander.Items>
+                            <controls:SettingsCard Header="{x:Bind LinkDescription}">
+                                <HyperlinkButton Content="{x:Bind Url}"
+                                                 NavigateUri="{x:Bind Url}" />
+                            </controls:SettingsCard>
+                        </controls:SettingsExpander.Items>
+                    </controls:SettingsExpander>
+                </Grid>
+            </DataTemplate>
+        </ListView.ItemTemplate>
+        <ListView.ItemContainerStyle>
+            <Style BasedOn="{StaticResource DefaultListViewItemStyle}"
+                   TargetType="ListViewItem">
+                <Setter Property="Margin" Value="0" />
+                <Setter Property="Padding" Value="4" />
+            </Style>
+        </ListView.ItemContainerStyle>
+        <!--  Hides the overall highlight from the ListView Container  -->
+        <ListView.Resources>
+            <SolidColorBrush x:Key="ListViewItemBackgroundPointerOver"
```

Comment:
When dragging starts, the item still shows a hightlight. We likely need to override `ListViewItemBackgroundPressed` to suppress that.

```xaml
<SolidColorBrush x:Key="ListViewItemBackgroundPressed" Color="Transparent" />
```

Not sure if `ListViewItemBackgroundPointerOver` has any effect in this context. It might be worth double-checking.

Reply:
> Just something simple still for now to show how it can be composited and made working with a behavior.

Bumping my comment here just in case...😅

```diff
-- <SolidColorBrush x:Key="ListViewItemBackgroundPointerOver " Color="Transparent" />
++ <SolidColorBrush x:Key="ListViewItemBackgroundPressed" Color="Transparent" /> 
```